docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Teleport with GitHub.
+description: How to use SSH key extensions with Teleport and GitHub.
+h1: Use SSH Key extensions.
+---
+
+## Prerequisites
+
+- Teleport v(=teleport.version) Open Source or Enterprise.
+
+## Step 1. Import the Teleport CA into Github
+
+In order to export the Teleport CA, execute
+
+```code
+tctl auth export > teleport.ca
+```
+
+Then follow the steps outlined[here](https://docs.github.com/en/organizations/managing-git-access-to-your-organizations-repositories/managing-your-organizations-ssh-certificate-authorities),
```

Comment:
Should we make it explicit that you would be pasting the contents of `teleport.ca` into the "Key" field in the "Add new" menu in GitHub?